app/Http/Controllers/Admin/AdminController.php

```diff
@@ -15,21 +17,160 @@ class AdminController extends Controller
      */
     public function index()
     {
-        return view('admin.index');
+        return view('admin.user.dashboard', ['roleAdmin' => $this->roleAdmin()]);
     }
 
+    /**
+     * Permisson account admin
+     *
+     * @return $roleAdmin
+     */
+    public function roleAdmin()
+    {
+        $roleAdmin = \App\User::ADMIN;
+        return $roleAdmin;
+    }
+
+    /**
+     * Display user list.
+     *
+     * @return Illuminate\View\View
+     */
     public function userList()
     {
-        return view('admin.user.list');
+        $users = Auth::user();
+        if ($users->role === \App\User::USER) {
+            return redirect()->back();
+        } else {
+            $users = User::where('role', \App\User::ROLE_USER)
+                ->paginate(\App\User::PAGINATE_USER);
+            return view(
+                'admin.user.list', [
+                'users' => $users, 'roleAdmin' => $this->roleAdmin()
+                ]
+            );
+        }
     }
 
+    /**
+     * Return view create.
+     *
+     * @return Illuminate\View\View
```

Comment:
Currently there are 2 type of writing about same item "@return Illuminate" and "@return \Illumintate" , please chose 1 and fix other

Reply:
Fixed, I choose "@return \Illumintate"